SleepEngine/Engine/Jobs/JobSystem.cpp

```diff
@@ -0,0 +1,86 @@
+#include "stdafx.h"
+#include "JobSystem.h"
+
+BEGIN_NAMESPACE_SLEEP
+
+namespace
+{
+    constexpr unsigned ThreadsLeftAlone = 1;

```

Comment:
'constexpr' is unneeded here.
Yes, it is not a mistake to use it, some people prefer to 'ALL THAT CAN BE CONSTEXPR SHOULD BE CONSTEXPR'.
But I found 'constexpr' less readable then simple 'const'.
Compiler will optimize runtime storage of this constant anyway.

Reply:
yeah lets stick with const